apps/docs/src/app/core/component-docs/avatar-group/examples/avatar-group-group-type-example.component.scss

```diff
@@ -23,3 +23,10 @@
         }
     }
 }
+
+.fd-avatar-group--group-type__popover-control--custom-class:focus {
+    outline-offset: 0.0625rem;
+    outline-width: 0.0625rem;
+    outline-color: #000;
+    outline-style: dotted;
+}
```

Comment:
You can't introduce a class like this in the documentation. 
1st - the naming is absurd!!! Please check the BEM guidelines
2nd - the problem is fixed only in fund-ngx documentation. If the user implements this component the problem will still be there.
3rd - in the documentation scss files you should only have classes that make the rendering of the components nice aka the documentation representation, you should not introduce or modify fundamental classes.